src/main/java/com/puppycrawl/tools/checkstyle/AbstractAutomaticBean.java

```diff
@@ -338,9 +341,10 @@ private static final class UriConverter implements Converter {
 
         @SuppressWarnings("unchecked")
         @Override
+        @Nonnull
```

Comment:
Why are we claiming that this method returns a non-null value when the return value can definitely be null?

Reply:
I made new changes @nrmancuso Please review this one. 

--> Method return type was @Nonnull and type of expression was @Nullable 
--> Overall i did @Nullable
--> In the end casted result to Object so to make  compatible return type